app/grandchallenge/profiles/models.py

```diff
@@ -44,6 +44,7 @@ class UserProfile(models.Model):
     department = models.CharField(max_length=100)
     country = CountryField()
     website = models.CharField(max_length=150, blank=True)
+    organization_display = models.BooleanField(default=False, blank=True)
```

Comment:
Could we call it "display_organizations" instead? And add some help text: "Display the organizations that you are a member of in your profile". Default should be `True`. Blank should be removed (either display it or do not).